src/Tuttifrutti/Cache/Storage.hs

```diff
@@ -62,4 +62,4 @@ lookupValid h k validate =
 
 -- | Drop elements whose priority is less than given value.
 dropLowerThan :: Handle k p v m -> p -> m ()
-dropLowerThan h = dropRange h . UpperBoundRange
+dropLowerThan h p =  dropRange h $ UpperBoundRange (Bound { boundValue = p, boundType = Inclusive })
```

Comment:
Isn't this and the following changes going to affect other repos? I saw some of these used in other projects.

Reply:
Yes, but we use commit hashes when pulling Tuttifrutti as a dependency so it shouldn't break anything (well of course the code needs to be changed where Tuttifrutti is updated)